lib/internal/bootstrap_node.js

```diff
@@ -227,6 +217,10 @@
             // User passed '-e' or '--eval'
             evalScript('[eval]');
           }
+
+        } else if (!process.allow_repl && NativeModule.require('tty').isatty(0)) {
+          console.log('Embedded mode');
+          evalScript('[eval]');
```

Comment:
Can we add the if-check from line 216 here? (`if (process._eval != null) { }`)

Reply:
I think @cmfcmf said that we **need** to do the evalScript call, otherwise the code won't run at all. Let's wait for his input?